input/fsh/instances/encounter-birth-babyg-quinn.fsh

```diff
@@ -14,10 +14,10 @@ Usage: #example
 * participant[0]
   * type = $loinc#87287-9 "Birth certifier [Extended Identifier]"
   * period.start = "2019-02-12T13:30:00-07:00"
-  * individual = Reference(practitioner-vital-records-avery-jones) "Practitioner - Vital Records (Avery Jones, MD)"
+  * individual = Reference(practitioner-vital-records-avery-jones) "Practitioner - Certifier (Avery Jones, MD)"
```

Comment:
The referenced practitioner hasn't changed, and indeed the practitioner should conform to:
1) Practitioner Vital Records ("base profile")
2) Certifier
3) Attendant

Bot the references point to the same instance, and I think the instance is still a Practitioner Vital Records, so I would undo the changes in the display name.